src/runtime/virtcontainers/clh.go

```diff
@@ -187,6 +187,12 @@ var clhDebugKernelParams = []Param{
 //
 //###########################################################
 
+func (clh *cloudHypervisor) supportsSharedFS() bool {
+	caps := clh.Capabilities(clh.ctx)
+
+	return caps.IsFsSharingSupported()
```

Comment:
nit: `return clh.Capabilities(clh.ctx).IsFsSharingSupported()`

Reply:
caps is a pointer and the compiler complains if we do that.  The cleaner way I found is this.